jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -12,14 +12,18 @@ export CAPM3_VERSION
 export IMAGE_OS
 export DEFAULT_HOSTS_MEMORY
 export NUM_NODES
-export KUBERNETES_VERSION
 
 if [ "${REPO_NAME}" == "airship-dev-tools" ]
 then
   export IMAGE_NAME
   export IMAGE_LOCATION
 fi
 
+if [ "${CAPM3_VERSION}" == "v1alpha4" ]
+then
+  export KUBERNETES_VERSION="v1.21.2"
```

Comment:
are not we using "v1.22.0" KUBERNETES_VERSION?

Reply:
for v1a4 no